### PR TITLE
Alex Martin PR

### PR DESCRIPTION
**Pull request by ghost**
@KingAtoki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,26 +45,26 @@ Upon your first commit, please submit a Pull Request and add _both_ the **Trello
 
 ## Deployment
 
-After your PM certifies that all MVP features are complente, it's time to work on deploying it online for the world to see. We recommend you deploy your server to [Heroku](https://devcenter.heroku.com/articles/getting-started-with-nodejs#introduction), and your front end to [netlify](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/). Both services provide free tiers and easy to use interfaces.
+After your PM certifies that all MVP features are complete, it's time to work on deploying it online for the world to see. We recommend you deploy your server to [Heroku](https://devcenter.heroku.com/articles/getting-started-with-nodejs#introduction), and your front end to [netlify](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/). Both services provide free tiers and easy to use interfaces.
```
Comment (KingAtoki) [inline README.md]: Nice lol

File: auth/authenticate.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  process.env.JWT_SECRET ||
+  'add a .env file to root of project with the JWT_SECRET variable';
+
+// quickly see what this file exports
+module.exports = {
+  authenticate,
+  generateToken
+};
+
+function generateToken(user) {
```
Comment (KingAtoki) [inline auth/authenticate.js]: If you are only going to use one thing from the the object that you are passing in then you can just destructure it as a parameter
```suggestion
function generateToken({ username }) {
```

File: auth/authenticate.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  const payload = {
+      username: user.username,
```
Comment (KingAtoki) [inline auth/authenticate.js]: Then you can just do 
```suggestion
      username,
```
Works both the same 👍

File: notes/notesModel.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+const db = require('../data/dbConfig.js');
+
+module.exports = {
+    get: function(id) {
+        if(id) {
+            return db('notes').where('id',id)
```
Comment (KingAtoki) [inline notes/notesModel.js]: I'm pretty sure you can just do `.first()` here instead of the `.then` statement below

**Comment by KingAtoki** [comment]
Nice job so far! Everything looks good. You code is clean and works as expected. Keep up the good work

File: auth/authenticate.js (hunk 3 above)
Comment (atonymartin20) [reply auth/authenticate.js]: I like it thank you sir 👍

File: notes/notesModel.js (hunk 4 above)
Comment (atonymartin20) [reply notes/notesModel.js]: Worked great.  Thank you

**Pull request closed without merging**